proto/tendermint/abci/types.proto

```diff
@@ -251,14 +251,10 @@ message ResponseBeginBlock {
 message ResponseCheckTx {
   uint32         code       = 1;
   bytes          data       = 2;
-  string         log        = 3;  // nondeterministic
-  string         info       = 4;  // nondeterministic
-  int64          gas_wanted = 5;
-  int64          gas_used   = 6;
-  repeated Event events     = 7 [(gogoproto.nullable) = false, (gogoproto.jsontag) = "events,omitempty"];
-  string         codespace  = 8;
-  string         sender     = 9;
-  int64          priority   = 10;
+  int64          gas_wanted = 3;
+  string         codespace  = 4;
+  string         sender     = 5;
+  int64          priority   = 6;
```

Comment:
We should probably also mark these fields reserved so we don't reuse the old field tags later. (Unlike the other ABCI++ changes, these fields have been around since beforehande)
```suggestion
  int64          priority   = 6;
  
  reserved 7, 8, 9, 10;  // see https://github.com/tendermint/tendermint/issues/8543
```

Reply:
In normal conditions, these fields should be first deprecated for at least one release, then removed; as they have been around and some apps might be filling them up (the App developer might not know these fields are not looked at by Tendermint).

However, given the important changes to the ABCI interface that ABCI++ is anyway introducing, deprecation of these fields is probably overkill in this case.

Reply:
Reserving the fields is orthogonal to deprecation, and reserving them is good for the future (though I agree that we don't need to deprecate because of the other changes.)